### PR TITLE
CON-2448 Asm - Amazon GuardDuty docs added, cleanup of github descriptions

### PR DESCRIPTION
**Pull request by jakub-jarek**

### CODE REVIEW

File: guides/applications/v7/security/asm.md [hunk 1]
```diff
@@ -9,16 +9,18 @@ title: Attack Surface Management
...
+-   **Assets:** Your assets consist of hostnames, IP addresses, GitHub repositories, Snyk targets, and AWS Resources. There are two methods for registering an asset.
+    -   An asset is registered for each hostname, IP address, GitHub repository defined as a seed.
+    -   {{ PRODUCT }} uses seeds to determine which network segments will be scanned. Each hostname and IP address identified through this scan is also registered as an asset. 
+        For more complex seeds that target whole organizations/systems, such as Snyk and Amazon GuardDuty, multiple assets (Snyk targets and AWS resources) are created as assets.
 -   **Exposures:** By default, {{ PRODUCT }} scans your network for:
     -   Common Vulnerabilities and Exposures (CVE). A CVE represents a known security vulnerability or exposure for a software package. 
     -   Common Weakness Enumeration (CWE). A CWE identifies a common software or hardware weakness that can potentially introduce a security vulnerability. 
     -   Open ports.
     -   TLS issues.
     -   Exposed secrets.
```
Comment (jsernaque) [inline guides/applications/v7/security/asm.md:21]:
```suggestion
```

**Review by jsernaque — CHANGES_REQUESTED** [review]
Please review these suggestions.

File: guides/applications/v7/security/asm.md [hunk 2]
```diff
@@ -9,16 +9,18 @@ title: Attack Surface Management
 Definitions for key concepts are provided below.
 
 -   **Collection:** A [collection](#collections) represents the segment(s) of your network that will be scanned for vulnerabilities. 
--   **Assets:** Your assets consist of hostnames, IP addresses, GitHub repositories, and Snyk instances. There are two methods for registering an asset.
-    -   An asset is registered for each hostname, IP address, GitHub repository, and Snyk instance defined as a seed.
-    -   {{ PRODUCT }} uses seeds to determine which network segments will be scanned. Each hostname and IP address identified through this scan is also registered as an asset.
+-   **Assets:** Your assets consist of hostnames, IP addresses, GitHub repositories, Snyk targets, and AWS Resources. There are two methods for registering an asset.
+    -   An asset is registered for each hostname, IP address, GitHub repository defined as a seed.
+    -   {{ PRODUCT }} uses seeds to determine which network segments will be scanned. Each hostname and IP address identified through this scan is also registered as an asset.
```
[the remark this answers is not in the document]
Comment (jakub-jarek) [reply guides/applications/v7/security/asm.md:14]: no - seeds are above ips and hostnames in this sense...eg. you might have seed for github repo, but that github repo will have connected hostnames etc.

File: guides/applications/v7/security/asm.md (hunk 1 above)
Comment (jakub-jarek) [reply guides/applications/v7/security/asm.md:21]: Why? Exposed secrets are targetable.

File: guides/applications/v7/security/asm.md [hunk 3]
```diff
@@ -9,16 +9,18 @@ title: Attack Surface Management
...
+    -   {{ PRODUCT }} uses seeds to determine which network segments will be scanned. Each hostname and IP address identified through this scan is also registered as an asset. 
+        For more complex seeds that target whole organizations/systems, such as Snyk and Amazon GuardDuty, multiple assets (Snyk targets and AWS resources) are created as assets.
 -   **Exposures:** By default, {{ PRODUCT }} scans your network for:
     -   Common Vulnerabilities and Exposures (CVE). A CVE represents a known security vulnerability or exposure for a software package. 
     -   Common Weakness Enumeration (CWE). A CWE identifies a common software or hardware weakness that can potentially introduce a security vulnerability. 
     -   Open ports.
     -   TLS issues.
     -   Exposed secrets.
     -   Response header violations. For example, a required header may be missing or it can assigned an invalid value.
+    -   AWS issues.
```
[the remark this answers is not in the document]
Comment (jakub-jarek) [reply guides/applications/v7/security/asm.md:22]: CVEs and CWEs are already part of snyk, so this feels strange

[the remark this answers is not in the document]
Comment (jakub-jarek) [reply guides/applications/v7/security/asm.md:22]: I also added back exposed secrets to match this

**Comment by jakub-jarek** [comment]
> Please review these suggestions.

Integrated most of it - there are 2 things which i didnt integrate though, at it changed meaning of things - described why underneath them

**Review by jsernaque — APPROVED** [review]

**Pull request merged**